internal/secrets/awskms/example_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awskms_test
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"gocloud.dev/internal/secrets/awskms"
+)
+
+func Example_encrypt() {
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String("us-west-1"),
+	})
+	if err != nil {
+		panic(err)
```

Comment:
Nit: Use log.Fatal instead of panic.

https://github.com/google/go-cloud/blob/master/internal/docs/design.md#coding-conventions